examples/addons/aad-pod-identity/README.md

```diff
@@ -41,10 +41,6 @@ This is the AAD Pod Identity add-on.  Add this add-on to your json file as shown
             }
           ]
         }
-      },
```

Comment:
Cleaning up these placeholder objects as we're no longer emphasizing the service principal solution